### DISCUSSION TITLE
помогите найти обход wikidot.com через zapret

### DISCUSSION THREAD

**Discussion by TIXefy** [bol-van/zapret#929] [Q&A]
просто мне надо юзать wikidot.com а не могу из-за блокировки
(помогите если конечно возможно)

**Comment by BubaMuba** [comment]
Юзайте через vpn с зарубежным ip. По-другому никак.

**Reply by ivanxmac** [reply]
Заворачивай в TOR запросы к wikidot.com. У меня на роутере keenetic для блокировок РКН nfqws крутится, для иностранных блокировок русских ip настроен обход через tor.
